metadata.rb

```diff
@@ -9,6 +9,6 @@
 version          '3.0.0'
 
 supports 'ubuntu', '>= 14.04'
+supports 'centos', '>= 6.6'
 
-depends 'apt', '~> 3.0.0'
```

Comment:
That's interesting that ubuntu no longer needs this. I'll have to remember that.